Foundation/NSStream.swift

```diff
@@ -58,22 +58,20 @@ extension Stream {
     }
 }
 
-public func ==(lhs: Stream.PropertyKey, rhs: Stream.PropertyKey) -> Bool {
```

Comment:
I know the discussion of this api is ongoing (and isn't related to this PR), but I think it would be better to not prevent implementation due to a non-finalized api. I know many of us would love to be able to use this functionality on Linux. Here's a simple version I threw together that might satisfy the use case.

If this PR is to be merged, I'm wondering if we could also add a quick implementation so we aren't blocked.

public class func getStreamsToHost(withName hostname: String, port: Int, inputStream: AutoreleasingUnsafeMutablePointer<InputStream?>?, outputStream: AutoreleasingUnsafeMutablePointer<NSOutputStream?>?) {
        var read: Unmanaged<CFReadStream>? = nil
        var write: Unmanaged<CFWriteStream>? = nil
        CFStreamCreatePairWithSocketToHost(kCFAllocatorDefault, host, UInt32(port), &read, &write)
        inputStream = read?.takeRetainedValue()
        outputStream = write?.takeRetainedValue()
    }